docs/products/caching/concepts/overview.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Aiven for Caching overview
+---
+
+Aiven for Caching, a fully managed **in-memory NoSQL database**, offers efficient storage and quick data access in your preferred cloud. Compatible with legacy Redis® OSS, this service facilitates seamless transitions and compatibility.
+
+With Aiven, you can leverage the power of this in-memory database to improve the
+performance of your applications by setting up high-performance data caching.
+Additionally, it can be integrated seamlessly into your observability stack for
+purposes such as logging and monitoring.
+
+Aiven for Caching supports a wide range of data structures, including strings, hashes,
+lists, sets, sorted sets with range queries, bitmaps, hyperloglogs,
+geospatial indexes, and streams.
+
+## Features and benefits of Aiven for Caching
+
+Aiven for Caching has many features that make it easy and stress-free to
+use:
+
+-   **Managed service:** Fully managed service, so you don't
+    have to worry about setup, management, or updates. Aiven provides
+    tools and integrations to help you use Redis in your data
```

Comment:
maybe we need to change the sentence to not include Redis?

Reply:
done.